posthog/api/test/test_decide.py

```diff
@@ -186,11 +186,6 @@ def test_user_console_log_opt_in(self, *args):
 
     def test_user_performance_opt_in(self, *args):
         # :TRICKY: Test for regression around caching
-        response = self._post_decide().json()
-        self.assertEqual(response["capturePerformance"], False)
-
-        self._update_team({"capture_performance_opt_in": True})
-
```

Comment:
this deletion means this test now only covers the default value, not that we can also change the value

i'm always torn on whether something like this is one test or two... but I think we should still cover that we can disable the setting and see the decide response change

Reply:
Agreed, making this change

Reply:
Resolved via commit: https://github.com/PostHog/posthog/pull/23900/commits/1ed2cda452ad08f17f7c53012269f14b339fe6b8